src/seedu/addressbook/AddressBook.java

```diff
@@ -202,7 +202,22 @@
 	public static void main(String[] args)
 	{
 		showWelcomeMessage();
-		processProgramArgs(args);
```

Comment:
inlining method calls decreases SLAP a lot. What else would? Decreasing SLAP will also make the codebase bulky. Note that inline doesn't always do that. As we discussed inlining mainly makes navigation and debugging difficult.